packages/aurora-note/view/test/note.test.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import NoteView from "../Note.js";
+import { EditorState, ContentState } from "draft-js";
+import { mount } from "enzyme";
+import sinon from "sinon";
+
+describe("note", () => {
+  it("should render a delete button and we can click it", () => {
+    const fakeDeleteFunc = sinon.spy();
+
+    const wrapper = mount(
+      <NoteView
+        defaultEditorState={EditorState.createWithContent(
```

Comment:
This creating an editor state from text could be a function somewhere. It's done a lot in tests.

Reply:
It really should be. I might add it in a separate thing though